public/api-docs/profile.yaml

```diff
@@ -0,0 +1,69 @@
+definitions:        # Schema defination for request body
+  profile:
+    type: object
+    required: 
+      - gender
+      - role
+      - birthdate
+    properties:
+      gender:
+        type: string
+      language:
+        type: string
+      currency:
+        type: string
+      location:
+        type: string
+        description: Where do you live
+      role:
+        type: string
```

Comment:
@SergeNm this should an integer not a string. Kindly pass through your documentation and write it well.